internal/database/sqliterepository.go

```diff
@@ -38,8 +38,6 @@ var keys = []keyDefinition{
 	{"release", "TEXT NOT NULL"},
 	{"prerelease", "INTEGER NOT NULL"},
 	{"kibanaVersion", "TEXT NOT NULL"},
-	{"capabilities", "TEXT NOT NULL"},
-	{"discoveryFields", "TEXT NOT NULL"},
```

Comment:
`discoveryFields` is not used currently as a filter, and it could have the same issues as `categories` and `capabilities`.
So I think it is better to remove it from here for now.